README.md

```diff
@@ -28,7 +28,7 @@ Simple PDF Generator:
 
 ## Quick Start
 
-In order to have a template you must create struct with `PdfTemplate` derive:
+In order to have a template you must create struct with `PdfTemplate` or `PdfTemplateForHtml` derive:
```

Comment:
You should even add an usage example and update the `test_suite` as well

Reply:
Makes sense! I will be sure to do that in due time